test/suite/extension.test.ts

```diff
@@ -16,7 +16,7 @@ describe("extension.ts", () => {
     expect(commands).to.include.members(["assay.get"]);
     expect(commands).to.include.members(["assay.welcome"]);
     expect(commands).to.include.members(["assay.review"]);
-    expect(context.subscriptions.length).to.equal(14);
+    expect(context.subscriptions.length).to.be.greaterThan(10);
```

Comment:
is this going to be variable, or did you just get bored of increasing it?

Reply:
the latter, I think that increasing it by one each time doesn't really contribute to the correctness of the test